drivers/go-getting-started.md

```diff
@@ -43,7 +95,7 @@ conn, err := http.NewConnection(http.ConnectionConfig{
 if err != nil {
     // Handle error
 }
-c, err := driver.NewClient(driver.ClientConfig{
+client, err := A.NewClient(driver.ClientConfig{
```

Comment:
IMHO changing from `driver` to `A` lowered the readability of this. UpperCase names in go should be reserved to exported functions, types, variables, and constants.